src/metabase/analytics/snowplow.clj

```diff
@@ -127,15 +128,29 @@
    ::timeline  "1-0-0"
    ::task      "1-0-0"})
 
+(defn app-db-type
+  "Returns the type of the Metabase application database as a string (e.g. PostgreSQL, MySQL)"
+  []
+  (clojure.java.jdbc/with-db-metadata [metadata (db/connection)]
```

Comment:
```suggestion
  (jdbc/with-db-metadata [metadata (db/connection)]
```

I know you're requiring `clojure.java.jdbc` above but it's easier to tell at a glance that you remembered to do it if you use the alias. If I just see something like `some.full.path.to.namespace/whatever` somewhere I always have to go look and make sure it's `require`d since otherwise it's a race condition and the code works if someone else has already required that namespace by the time we got here vs not requiring it. I guess Kondo checks that these days but it still makes me itchy. I guess if for no other reason than consistency it's probably best to use the alias

Reply:
yep, I forgot to change these to the alias, thanks for catching